Pull_Request_and_Code_Review_Standards.md

```diff
@@ -0,0 +1,49 @@
+# Pull Request and Code Review Standards
+
+-   Unless it's a trivial change (spelling mistake, code formatting, etc.), there should be an existing issue that will be closed by the given PR.
+
+-   The branch name from which PR is being made should follow the convention: `\<issue_number\>-short-description`.
+
+-   The title for the PR should provide a brief description about the changes.
+
+-   The PR should be linked to the issue(s) that it targets, either by using [keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) (like `fixes`, `closes`, etc.) in the PR description or by using `development` feature.
+
+    ![](figures/development.png)
+
+-   Even if the PR doesn't close any issues, it should refer to the comment or discussion that motivated it. Lacking any of these, the PR description should be self-sufficient to explain the motivation for the PR.
+
+-   If the PR is a work in progress, you can convert it to a draft PR and add `WIP` to the PR title.
+
+    ![](figures/draft.png)
+
+-   Reviewers should not conduct code review of draft PRs, unless otherwise requested by the contributors.
+
+-   Once the PR is marked as "Ready for Review", contributors should remember to remove `WIP` from the PR title.
+
+-   Before marking the PR as "Ready for Review", contributors should also try their best to make sure that they are following the organization's [coding standards](https://github.com/Open-Systems-Pharmacology/Suite/blob/develop/CODING_STANDARDS.md) and to make sure that CI/CD pipelines are executing successfully. If this is not possible for some reason (e.g. because the code in *other repositories* must be changed) - the contributor should create new issue(s) for those required code changes, link them in the PR, and shortly explain why the change in the PR itself is not possible.
+
+-   Ideally, there should be a one-to-one mapping of issues to PR, although if a number of issues are related to the same underlying functionality, it is completely acceptable to address multiple issues in a single PR.
+
+-   If it's a PR that targets multiple issues or adds significant new functionality, there might be multiple rounds of code reviews and multiple tasks to be completed. In such cases, try to prepare a checklist in the PR description and keep it updated so that the reviewers and contributors always have an up-to-date view of the status of the PR.
+
+-   Reviewers should do their due diligence to understand the underlying issue, instead of asking contributors to reiterate the details that have already been outlined in GitHub Issues/Discussions.
+
+-   Reviewers should refrain from rubber-stamping reviews. If they feel unqualified or do not have time to review, they should urge contributors to request review from other collaborators instead of rubber-stamping an approval.
```

Comment:
Remove this. paragraph. This does not mean anything in our context. we need to focus here on what's important and  not what may happens in any OS project